src/main/scala/org/broadinstitute/hail/driver/Main.scala

```diff
@@ -35,8 +35,12 @@ object SparkManager {
 
       conf.set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
       _sc = new SparkContext(conf)
-      _sc.hadoopConfiguration.set("io.compression.codecs",
-        "org.apache.hadoop.io.compress.DefaultCodec,org.broadinstitute.hail.io.compress.BGzipCodec,org.apache.hadoop.io.compress.GzipCodec")
+      if(gzAsBgz)
+        _sc.hadoopConfiguration.set("io.compression.codecs",
+          "org.apache.hadoop.io.compress.DefaultCodec,org.broadinstitute.hail.io.compress.BGzipCodec, org.broadinstitute.hail.io.compress.BGzipCodecGZ")
```

Comment:
How does this handle gzipped files that are actually gzipped?  Suppose that you import a gz (bgz) vcf and then annotate with a small tsv.gz (actually gz)